persistence/test/account_test.go

```diff
@@ -15,8 +15,8 @@ import (
 
 func FuzzAccountAmount(f *testing.F) {
 	db := persistence.PostgresContext{
-		Height:     0,
-		PostgresDB: testPostgresDB,
+		Height: 0,
+		DB:     *PostgresDB,
```

Comment:
Why did we switch this back from `testPostgresDB` to `PostgresDB`.

My reasoning for the other was two-fold:

1. It doesn't need to be exposed outside the test module -> hence lowercase.
2. It's a database used for testing, so I made the clear & explicit.